WordPress/src/main/java/org/wordpress/android/push/GCMMessageService.java

```diff
@@ -312,36 +312,12 @@ private static void bumpPushNotificationsAnalytics(Stat stat, Bundle noteBundle,
         }
     }
 
-    private static boolean canAddActionsToNotifications(Context context) {
```

Comment:
Would you mind to run code inspector on this file, and clean unused imports and declaration redundancy?